xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -489,6 +489,14 @@ $displayName##
  *#
 #macro(livetablecallback $divid $collist $colprops $transprefix)
 function (row, i, table) {
+   ## Make sure the "doc.location" property output is HTML in case the collist contains "doc.location"
+   #if($collist != $NULL && $collist.indexOf("doc.location") >=0)
```

Comment:
I would have used

    #if ($collist.contains('doc.location'))

Note that it's recommended to use single quotes when the text doesn't contain Velocity code. And there's a white space missing between ``#if`` and ``(`` (to be consistent with the code you wrote below).